%YAML 1.2
---
yolo_decoder_node:
  ros__parameters:
    conf_thres: 0.25
    iou_thres: 0.45
    max_det: 1000